README.md

```diff
@@ -21,57 +22,71 @@ Below are the steps outlined in detail for getting started along with various ti
+| **max_nautobot_version** | The maximum supported Nautobot version |
+| **nautobot_version** | Used for development purposes to decide with Nautobot-dev Docker image to use for development |
+| **camel_name** | Used to define the plugin's subclassing of `PluginConfig`, e.g. `MyPluginConfig(PluginConfig):` |
+| **project_short_description** | Used in the **description** field within `PluginConfig` |
+| **version** | Version of the new Nautobot plugin |
+| **Select open_source_license** | Determine if project is open source or not |
+| **docs_base_url**| The main URL where the project documentation will be hosted. For open source projects use the default (`https://docs.nautobot.com`). |
+| **docs_app_url**| The full URL for documentation hosting. You might want to shorten the project alias, for example `https://docs.nautobot.com/projects/chatops-my-plugin/en/latest` instead of `https://docs.nautobot.com/projects/nautobot-plugin-data-validation/en/latest`. Make sure there's no trailing `/`! |
 | **setup_local_mattermost_dev_env**  | Setup Local mattermost development environment flag                                             |
 
-> NOTE: Cookiecutter by default bakes the new cookie within the current working directory. If that is not desirable then use the `-o` option to specify a different output directory.
+> NOTE: Cookiecutter by default bakes the new cookie within the current working directory. If that is not desirable then use the `-o` option to specify a different output folder.
 
 ```bash
 ❯ cookiecutter .
 
-full_name [John Doe]: email [mail@example.com]: 
+codeowner_github_usernames [@smith-ntc]:
+full_name [Network to Code, LLC]:
+email [info@networktocode.com]:
+github_org [nautobot]:
 chatops_interactive_command [my_plugin]: 
 plugin_name [nautobot_plugin_chatops_my_plugin]: 
 verbose_name [Nautobot Plugin Chatops My Plugin]: 
 plugin_slug [nautobot-plugin-chatops-my-plugin]: 
 project_slug [nautobot-plugin-chatops-my-plugin]: 
-min_nautobot_version [1.2.0]: 
-max_nautobot_version [1.9999]: 
-nautobot_version [latest]: 
+repo_url [https://github.com/nautobot/nautobot-plugin-chatops-my-plugin]:
+base_url [chatops-my-plugin]:
+min_nautobot_version [1.5.2]:
+max_nautobot_version [1.9999]:
+nautobot_version [latest]:
 camel_name [NautobotPluginChatopsMyPlugin]: 
 project_short_description [Nautobot Plugin Chatops My Plugin]: 
-version [0.1.0]: 
+version [0.1.0]:
 Select open_source_license:
 1 - Apache-2.0
 2 - Not open source
-Choose from 1, 2 [1]: 
+Choose from 1, 2 [1]:
+docs_base_url [https://docs.nautobot.com]:
+docs_app_url [https://docs.nautobot.com/projects/chatops-my-plugin/en/latest]: https://docs.nautobot.com/projects/chatops-my-plugin/en/latest
 Select setup_local_mattermost_dev_env:
 1 - Yes
 2 - No
 Choose from 1, 2 [1]: 
 
-Congratulations!  Your cookie has now been baked.
+Congratulations!  Your cookie has now been baked. It is located at /vagrant/nautobot-plugin-chatops-my-plugin.
```

Comment:
```suggestion
Congratulations!  Your cookie has now been baked. It is located at /vagrant/nautobot-plugin-chatops-my-plugin
```

Reply:
Removes the period to help with double-click, copy and paste.

Reply:
Fixed by moving the path to the next line, however this goes against this PR target, to be in sync with the cookiecutter-ntc plugin. Will introduce this change there as well.